modules/discovery/info_disclosure/interesting_files.py

```diff
@@ -58,6 +59,10 @@ def module_run(self, hosts):
         ]
         count = 0
         for host in hosts:
+            if self.options['files']:
+                filenames = self.options['files'].split()
+                filetypes = [(fname, False) for fname in filenames]
```

Comment:
By setting the `verify` part of the tuple to `False` here, you ensure that the file is never reported as found because it will never make it into the block of code that starts on line 79. Did you not experience this when testing?

Reply:
apologies, I skimmed over this section of code too quickly late last night and thought the download was happening in a separate place. Definitely understand the importance of verification given that redirects would always report as "found".

I will revisit and go with an external file as you mentioned above (probly just a CSV in order to specify the verification string). Will mark this as a draft and ping you when I have a proper version. Thanks for taking a look and for comments and sorry for the oversight here

Reply:
resolved with the following commit:
https://github.com/lanmaster53/recon-ng-marketplace/pull/214/commits/3b8f9e02d6a16deef67c7bc01d1afdd006e2c9a5